nexus/src/app/background/tasks/support_bundle_collector.rs

```diff
@@ -558,6 +568,13 @@ impl BundleCollection {
         )
         .await?;
 
+        let sp_dumps_dir = dir.path().join("sp_task_dumps");
+        tokio::fs::create_dir_all(&sp_dumps_dir).await.with_context(|| {
+            format!("failed to create SP task dump directory {sp_dumps_dir}")
+        })?;
+        let sp_dumps_fut =
```

Comment:
We should strongly consider modifying `SupportBundleCollectionReport` in `nexus/types/src/internal_api/background.rs` to identify whether or not collection of the SP dumps was successful or not.